tests/Unit/Form/Eloquent/SharpFormEloquentUpdaterTest.php

```diff
@@ -82,6 +83,33 @@ public function update($id, array $data)
     ]);
 });
 
+it('HTML fields are ignored', function () {
```

Comment:
Please respect the standards: test title must be readable, starting with "it". So maybe 
`it('ignores SharpFormHtmlField fields')`